codex-rs/core/src/protocol.rs

```diff
@@ -33,6 +33,8 @@ pub struct Submission {
 #[allow(clippy::large_enum_variant)]
 #[non_exhaustive]
 pub enum Op {
+    /// Erase all conversation history for the current session.
```

Comment:
For a large enum, I generally try to put the most commonly used variants, or the ones that are likely to be used first (like `ConfigureSession`) at the top of the list, so I would list `EraseConversationHistory` later in this declaration.